src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -443,4 +448,18 @@ private void putCustomCapabilities(Object keyCustomCapabilities) {
     private void putCustomCapabilities(Map<String, Object> customCapabilities) {
         this.customCapabilities.putAll(customCapabilities);
     }
+
+    public SauceOnDemandAuthentication getAuthentication() {
+        String sauceUserName = System.getProperty("SAUCE_USERNAME");
+        String sauceAccessKey = System.getProperty("SAUCE_ACCESS_KEY");
```

Comment:
We should make these actual config fields that people can fill out and not just rely on the environment variable.

...Things I should've done yesterday but didn't haha

Reply:
done. think I fixed it proper but see next commit